assets/i18n/en.json

```diff
@@ -147,6 +147,7 @@
     "I18N_CREATE_ACTIVITY_TITLE": "Create an Activity",
     "I18N_CREATE_COLLECTION": "Create Collection",
     "I18N_CREATE_EXPLORATION": "Create Exploration",
+    "I18N_CREATE_EXPLORATION_BUTTON": "+ Create Exploration",
```

Comment:
Can we use '+' icon in the html instead of adding it here ?

Reply:
Done!